starter-code/public/javascripts/financial-data.js

```diff
@@ -0,0 +1,53 @@
+$(document).ready(() => {
+
+  const baseURL = "http://api.coindesk.com/v1/bpi/historical/close.json";
+
+  function setDate() {
+    $('button').on('click', function(){
+      console.log("Holi, funciono");
```

Comment:
"holi" tienes que hacer log de las variables from y to para enviarlos a la api y que os haga la actualizacion correspondiente del grafico